### PR TITLE
online_status: use helpers

### PR DESCRIPTION
**Pull request by lasers**
Reviving an issue that was closed, abandoned, left for dead, and almost forgotten: https://github.com/ultrabug/py3status/pull/696.

### CODE REVIEW

File: py3status/modules/online_status.py [hunk 1]
```diff
@@ -3,30 +3,29 @@
...
-    format: display format for online_status (default '{icon}')
-    icon_off: what to display when offline (default '■')
-    icon_on: what to display when online (default '●')
-    timeout: how long before deciding we're offline (default 2)
-    url: connect to this url to check the connection status
+    cache_timeout: refresh interval for this module (default 10)
+    format: display format for this module (default '{icon}')
+    icon_off: show when connection is offline (default '■')
+    icon_on: show when connection is online (default '●')
+    timeout: number of seconds before announcing offline (default 2)
```
Comment (tobes) [inline py3status/modules/online_status.py]: I think the old description is a bit better because it is the timeout rather than when we will 'announce'

Comment (lasers) [inline py3status/modules/online_status.py]: I bring back old description. Maybe we do `number of seconds before issuing offline`?

```
    timeout: how long before deciding we're offline (default 2)
    timeout: number of seconds before announcing offline (default 2)
    timeout: number of seconds before issuing offline (default 2)
```

Comment (tobes) [inline py3status/modules/online_status.py]: max time looking for network connection?

Comment (lasers) [inline py3status/modules/online_status.py]: `timeout: time to wait for a response, in seconds.` ?
`timeout: wait in seconds before deciding offline`  ?
`timeout: seconds to wait before deciding offline` ?
`timeout: specify a timeout, in seconds, before deciding offline` ?

Comment (tobes) [inline py3status/modules/online_status.py]: time to wait for a response, in seconds.  Is nicest I think - thanks

**Comment by tobes** [comment]
@lasers nice one.  We can probably use `py3.request()` in this module too.

**Pull request merged**